README.md

```diff
@@ -1,32 +1,504 @@
+    [[Swrve sharedInstance].talk pushNotificationReceived:userInfo];
+}
+```
+
+### Creating and uploading your iOS Certificate
+
+To enable your app to send push notifications to iOS devices, you require a push certificate. A push certificate authorizes an app to receive push notifications and authorizes a service to send push notifications to an app. For more information, see [How Do I Manage iOS Push Certificates?](http://docs.swrve.com/faqs/push-notifications/manage-ios-push-certificates-for-push-notifications/)
+
+### Provisioning your app
+
+Each time your app is published for distribution, you must provision it against the push certificate that has been created for the app. To do this, perform the following steps:
+
+1. In Xcode, on the Product menu, click Archive.
+
+2. On the Organizer, select Archives and then select the latest archive you want to distribute.
+
+3. Select the distribution for ad-hoc deployment.
+
+4. On the provisioning screen, select the development push certificate created for this app.
+
+
+
+Sending Events
+-
+
+### Sending Named Events ###
+
+```
+[[Swrve sharedInstance] event:@"custom.event_name"];
+```
+
+Rules for sending events:
+
+* Do not send the same named event in differing case.
+
+* Use '.'s in your event name to organize their layout in the Swrve dashboard. Each '.' creates a new tree in the UI which groups your events so they are easy to locate.
+* Do not send more than 1000 unique named events.
+ * Do not add unique identifiers to event names. For example, Tutorial.Start.ServerID-ABDCEFG
+ * Do not add timestamps to event names. For example, Tutorial.Start.1454458885
+* When creating custom events, do not use the `swrve.*` or `Swrve.*` namespace for your own events. This is reserved for Swrve use only. Custom event names beginning with `Swrve.` are restricted and cannot be sent.
+
+### Event Payloads ###
+
+An event payload can be added and sent with every event. This allows for more detailed reporting around events and funnels. The associated payload should be a dictionary of key/value pairs; it is restricted to string and integer keys and values. There is a maximum cardinality of 500 key-value pairs for this payload per event. This parameter is optional.
+
+```
+[[Swrve sharedInstance] event:@"custom.event_name" payload:@{
+  @"key1": @"value1",
+  @"key2": @"value2"
+}];
+```
+
+For example, if you want to track when a user starts the tutorial experience it might make sense to send an event `tutorial.start` and add a payload `time` which captures how long the user spent starting the tutorial.
+
+```
+[[Swrve sharedInstance] event:@"tutorial.start" payload:@{
+  @"time": @"100"
+}];
+```
+
+### Send User Properties ###
+
+Assign user properties to send the status of the user. For example create a custom user property called `premium`, and then target non-premium users and premium users in the dashboard..
```

Comment:
maybe mention here that user properties are based on the current status of the user, and anything sent will overwrite any previous values